web/app/view/UsersController.js

```diff
@@ -40,6 +40,12 @@ Ext.define('Traccar.view.UsersController', {
     onAddClick: function () {
         var user, dialog;
         user = Ext.create('Traccar.model.User');
+        if (Traccar.app.getUser().get('admin')) {
+            user.set('deviceLimit', -1);
```

Comment:
Added just not break old experiences, new user created admin will have infinite device limit by default